docs/maintainers/upgrading.md

```diff
@@ -1,5 +1,55 @@
 # Upgrading play-frontend-hmrc
 
+## Warning
+
+When you bump the govuk-frontend version, you need to manually check release changes for new components and new parameters that need to be added to our twirl implementations. They won't currently be automatically detected as missing by our automated tests. Read [comparing differences](#comparing-differences) below for ways you can do this.
```

Comment:
If we format it like this:
```suggestion
> **Warning**
> When you bump the govuk-frontend version, you need to manually check release changes for new components and new parameters that need to be added to our twirl implementations. They won't currently be automatically detected as missing by our automated tests. Read [comparing differences](#comparing-differences) below for ways you can do this.
```
then it will get some fancy styling like this https://github.com/community/community/discussions/16925
<img width="526" alt="CleanShot 2023-01-06 at 13 08 03@2x" src="https://user-images.githubusercontent.com/100650/211018363-bf6b4b55-89db-4715-8fe6-7a66e790636c.png">

Reply:
just to differentiate it a bit from the other writing on the page

Reply:
Great, i have learned something new here today